importer/specobject/src/main/java/org/itsallcode/openfasttrace/importer/specobject/SpecobjectImporterFactory.java

```diff
@@ -1,14 +1,22 @@
 package org.itsallcode.openfasttrace.importer.specobject;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Locale;
+import java.util.logging.Logger;
+
 import org.itsallcode.openfasttrace.api.importer.*;
 import org.itsallcode.openfasttrace.api.importer.input.InputFile;
 import org.itsallcode.openfasttrace.importer.xmlparser.XmlParserFactory;
 
 /**
- * An {@link ImporterFactory} for XML specobject files.
+ * An {@link ImporterFactory} for ReqM2/SpecObject XML files.
  */
 public class SpecobjectImporterFactory extends RegexMatchingImporterFactory
```

Comment:
Now that we have some custom logic in `supportsFile()`, we should extend `ImporterFactory` instead of `RegexMatchingImporterFactory`.

Reply:
Done.